### PR TITLE
Support Vector Machine Added with Sklearn

### LINKED ISSUE #76: Create a SVM example with scikit-learn
In #68 the [Breast Cancer Wisconsin (Diagnostic) Dataset](https://github.com/Rishit-dagli/Python3-by-practice/tree/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/Breast%20Cancer%20Wisconsin%20(Diagnostic)%20Dataset) as a sample Logistic Regression dataset was added by @Avinashshah099 . I am looking for contributors who could add a notebook with an aim to show how Logistic Regression works with `sklearn` to predict `diagnosis`. At the start of the PR description include `Closes #76` for automatic issue linking.

The notebook is expected to be placed under `Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb`

### Include Author name

You would be expected to include under the heading:

```
_Originally Contributed by [GitHub-Username](https://github.com/GitHub-Username)_
```

### Include buttons for accessibility
Include a `Run in Google Colab` and `View Source on GitHub` buttons

#### Example (taken from the [Linear Regression Notebook](https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb) in this repo)-

_Notebook orignially contributed by: [Aditya-1500](https://github.com/Aditya-1500)_

<table class="tfo-notebook-buttons" align="left">
  <td>
    <a target="_blank" href="https://colab.research.google.com/github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb">Run in Google Colab</a>
  </td>
  <td>
    <a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb">View source on GitHub</a>
  </td>
</table>

<br>
<br>
<br>

#### Code to generate buttons

```
<table class="tfo-notebook-buttons" align="left">
  <td>
    <a target="_blank" href="https://colab.research.google.com/github/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/SVM.ipynb"><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run in Google Colab</a>
  </td>
  <td>
    <a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/SVM.ipynb"><img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" />View source on GitHub</a>
  </td>
</table>
```
### Include a short intro

Below the above-mentioned notebook buttons, be sure to add a small intro showing what the notebook does.

### Include clear Dataset Credits

In the notebook, intro be sure to add clear credits to the dataset owner.

### PR DESCRIPTION
**Pull request by taneemishere**
Closes #76  
The SVM implementation with the Sklearn is being done here using the breast cancer dataset provided by you.
HACKTOBERFEST!

### CODE REVIEW

**Comment by Rishit-dagli** [comment]
@taneemishere Please follow all the guidelines in #76 . Also, the notebook is currently raw and has no explanations so as to what is happening, you should include explanations to help people understand. I will perform a detailed review soon.

**Comment by taneemishere** [comment]
Explanation added...

**Comment by Rishit-dagli** [comment]
@Avinashshah099 created #83 on the same lines and has included a lot of explanations for his notebook, would you like to co-author and improve this PR?

File: Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb [hunk 1]
```diff
@@ -0,0 +1,1041 @@
+{
```
Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb:1]: <p>Please use this code to create the buttons, I am well aware that these links will only work after the merge:</p><p> </p><pre class="ql-syntax" spellcheck="false">&lt;table class="tfo-notebook-buttons" align="left"&gt;
  &lt;td&gt;
    &lt;a target="_blank" href="https://colab.research.google.com/github/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/SVM.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/colab_logo_32px.png" /&gt;Run in Google Colab&lt;/a&gt;
  &lt;/td&gt;
  &lt;td&gt;
    &lt;a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Support%20Vector%20Machines(SVM)/SVM.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" /&gt;View source on GitHub&lt;/a&gt;
  &lt;/td&gt;
&lt;/table&gt;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb:1]: <p>Instead write (fixed typos and styling):</p><p> </p><blockquote>The following notebook uses a breast cancer dataset from USI machine learning repository that focuses on whether after seeing certain features, a patient has what type of cancer, i-e malignant or benign. &nbsp;It is followed by a series of steps involving data reading the analyzing, applying Machine Learning Algorithm the SVM Classifier and then printing out the accuracy score of this classifier. </blockquote><blockquote>We use the <a href="https://archive.ics.uci.edu/ml/datasets/Breast+Cancer+Wisconsin+(Diagnostic)" target="_blank">Breast Cancer Dataset</a></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb:1]: <p>Do not write "Nick Street" on the next line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb:1]: <p>Please remove "3-32)", looks like a typo to me.</p><p> </p><p>In formulas for compactness and fractal dimension, for proper rendering, use LaTeX</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb:1]: <p>This applies for all subsequent headings, they are not subheadings of Dataset Credit so please use second level headings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb:1]: <p>Could you shift this right under the the imports section to have all imports at the start.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb:1]: <p>Could you here explain in brief what other types of <code>kernel</code>  are available with <code>sklearn</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (Rishit-dagli) [inline Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb:1]: <p>Could you also have a conclusion part as in #83 to explain the readers that one of the accuracies is the train accuracy, maybe @Avinashshah099&nbsp;could help in this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

**Comment by Avinashshah099** [comment]
What kind of changes do you need in my notebook?? Should, I need to add more information about Kernels or something else??

**Comment by Rishit-dagli** [comment]
> What kind of changes do you need in my notebook?? Should, I need to add more information about Kernels or something else??

@Avinashshah099 I meant that since this PR was created earlier addressing the same issue, there has been a duplicate so I was wondering if you would like to make this PR better and co-author it?

**Comment by Avinashshah099** [comment]
@taneemishere I have gone through your notebook. And  I found something is missing, So can you add EDA and bit more of explanations on the step you have taken, confusion matrix  & conclusion

**Comment by Avinashshah099** [comment]
@taneemishere I have gone through your notebook. And  I found something is missing, So can you add EDA and bit more of explanations on the step you have taken, confusion matrix  & conclusion
